migrations/src/main/scala/za/co/absa/enceladus/migrations/continuous/migrate01/EntityMigrator.scala

```diff
@@ -57,17 +60,23 @@ abstract class EntityMigrator(databaseOld: MongoDatabase,
   def migrate(): Unit = {
     val repoOld = new EntityRepository(dbOld, collectionOld)
     val repoNew = new EntityRepository(dbNew, collectionNew)
+    var count = 0
+    var migratedCount = 0
 
     val entitiesOld = repoOld.getSortedDocuments
 
     entitiesOld.foreach(entityOld => {
       val objectId = ObjectIdTools.getObjectIdFromDocument(entityOld)
       objectId.foreach(id => {
+        count += 1
         if (!repoNew.doesDocumentExist(id)) {
+          migratedCount += 1
           migrateEntity(entityOld, id, repoNew)
         }
       })
     })
+
+    log.info(s"Migrated $migratedCount of $count entities from $collectionOld to $collectionNew.")
```

Comment:
Getting message Migrated 10 out of 100 entities might scare a person. But in this case, it means we found just 10 new entities in the old DB, right?